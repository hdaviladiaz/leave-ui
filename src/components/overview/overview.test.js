import React from 'react';
import Overview from './overview';
import renderer from 'react-test-renderer';

xit('should render Overview as snapshot', () => {
  const tree = renderer.create(<Overview/>).toJSON();
  expect(tree).toMatchSnapshot();
});
