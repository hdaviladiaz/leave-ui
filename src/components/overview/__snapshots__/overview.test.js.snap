// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Overview as snapshot 1`] = `
<div
  className="panel panel-default"
  id={undefined}
>
  <div
    className="panel-body"
  >
    <div
      className="overview"
    >
      <div
        className="col-xs-12"
      >
        <h4>
          21 junio 2017
        </h4>
        <span
          className="text-muted"
        >
           dias disponibles
        </span>
      </div>
      <div
        className="overview-box-container no-padding col-xs-12"
      >
        <div
          className="overview-box col-md-4"
        >
          <div
            className="overview-box-header gray-color"
          >
            Días Disponibles
          </div>
          <div
            className="overview-box-body"
          >
            <div
              className="overview-box-title"
            />
          </div>
        </div>
        <div
          className="overview-box col-md-4"
        >
          <div
            className="overview-box-header"
          >
            Última solicitud
          </div>
          <div
            className="overview-box-body"
          >
            <div
              className="overview-box-title"
            >
              n/a
            </div>
            <div
              className="text-muted"
            />
          </div>
        </div>
        <div
          className="overview-box last overview-box-center col-md-4"
        >
          <a
            className="btn btn-primary"
          >
            Solicitar Ahora
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
